evaluator/src/test/kotlin/EvaluatorTest.kt

```diff
@@ -53,24 +56,49 @@ class EvaluatorTest : WordSpec() {
             "return no errors for an empty script" {
                 val result = Evaluator(OrtResult.EMPTY).run("")
 
-                result.errors should beEmpty()
+                result.violations should beEmpty()
             }
 
             "contain rule errors in the result" {
                 val result = Evaluator(OrtResult.EMPTY).run(
                     """
-                    evalErrors += OrtIssue(source = "source 1", message = "message 1")
-                    evalErrors += OrtIssue(source = "source 2", message = "message 2")
+                    ruleViolations += RuleViolation(
+                        rule = "rule 1",
+                        pkg = Identifier("type:namespace:name:1.0"),
+                        license = "license 1",
+                        licenseSource = LicenseSource.DETECTED,
+                        severity = Severity.ERROR,
+                        message = "message 1"
+                    )
+
+                    ruleViolations += RuleViolation(
+                        rule = "rule 2",
+                        pkg = Identifier("type:namespace:name:2.0"),
+                        license = "license 2",
+                        licenseSource = LicenseSource.DECLARED,
+                        severity = Severity.WARNING,
+                        message = "message 2"
+                    )
                     """.trimIndent()
                 )
 
-                result.errors should haveSize(2)
-                result.errors[0].let {
-                    it.source shouldBe "source 1"
+                result.violations should haveSize(2)
+
+                result.violations[0].let {
+                    it.rule shouldBe "rule 1"
+                    it.pkg shouldBe Identifier("type:namespace:name:1.0")
+                    it.license shouldBe "license 1"
+                    it.licenseSource shouldBe LicenseSource.DETECTED
+                    it.severity shouldBe Severity.ERROR
                     it.message shouldBe "message 1"
                 }
-                result.errors[1].let {
-                    it.source shouldBe "source 2"
+
+                result.violations[1].let {
```

Comment:
Nit (can be done later): Maybe better use `with(result.violations[1])` here?

Reply:
Done.